manager-bundle/src/HttpKernel/ContaoCache.php

```diff
@@ -51,48 +49,18 @@ public function __construct(ContaoKernel $kernel, string|null $cacheDir = null)
         $this->addSubscriber(new CleanupCacheTagsListener());
     }
 
-    /**
-     * Overwrites the getEventDispatcher() method of the EventDispatchingHttpCache
-     * trait, so the LegacyEventDispatcherProxy is not used. Once we have upgraded
-     * to Symfony 6, the method can be removed again.
-     */
-    public function getEventDispatcher(): EventDispatcher
-    {
-        return $this->eventDispatcher ??= new EventDispatcher();
-    }
-
     public function fetch(Request $request, $catch = false): Response
     {
         return parent::fetch($request, $catch);
     }
 
-    /**
-     * Override default terminate method in order to never call the
-     * "kernel.terminate" event on cache hit.
-     *
-     * @todo Remove once symfony/http-kernel is required in at least ^6.2
-     */
-    public function terminate(Request $request, Response $response): void
-    {
-        $traces = $this->getTraces();
-
-        if (\in_array('fresh', $traces[$this->getTraceKey($request)] ?? [], true)) {
-            return;
-        }
-
-        $kernel = $this->getKernel();
-
-        if ($kernel instanceof TerminableInterface) {
-            $kernel->terminate($request, $response);
-        }
-    }
-
     protected function getOptions(): array
     {
         $options = parent::getOptions();
 
         $options['trace_level'] = $_SERVER['TRACE_LEVEL'] ?? 'short';
         $options['trace_header'] = 'Contao-Cache';
+        $options['terminate_on_cache_hit'] = false;
```

Comment:
_Note:_  Symfony 6 still executes `teminate` by default. However, leaving the default - or setting this to `true` is deprecated and will be `false` by default in Symfony 7.

Reply:
Then we should add that as a comment like we did for the other stuff 😊

Reply:
Done 👍